### ISSUE TITLE
[Project 4][Review][Feedback]Farm2Home

### ISSUE THREAD

**Issue by Harshi7016** [tanaypratap/teamtanay.jobchallenge.dev#864]
_description_  An eCommerce website for vegetables, the focus was mainly on creating E2E application and I was really happy looking at the end project. The project has signing, register, listing, cart, payment process steps. Through this project, I got the opportunity to explore more on backend NodeJS, ExpressJS, and MongoDB. The hosting of the project is done in the Heroku Platform.

_stack_ React,NodeJS,ExpressJS and MongoDB
_platform_ Heroku

_hosted link_ https://farm2wohome.herokuapp.com/

_github link_ https://github.com/Harshi7016/Farm2Homev1

Please review the project. Kanishk thanks for the update in Discord.
Mentors please review the project and do give constructive feedback.
I am still working on the search and sorting functionality part and will update soon.
All these aside, Great job guys for taking the time and reviewing our code and building this dev community.

**Comment by reeversedev** [comment]
Hey Harshith, I really wanna review your project but I think there's a lot of improvement in terms of UI/UX. Try fixing basic things like Carousel Height and Width properly so that images are not getting shrink.

**Comment by reeversedev** [comment]
I think the project is not completed right now, please take your time and then submit with your final draft. I'll help you if you need any help.

**Comment by Harshi7016** [comment]
Hey 
Thanks for the update.
Yeah, I do agree it is not complete yet and UI has a lot of scope in terms of improvement.
But I spent a lot of time on this project due to which I couldn't concentrate on the rest of the projects.
But yeah I will change the background picture and align the width and height of carousel right away

**Comment by reeversedev** [comment]
Here's my suggestion, divide your project into different modules. 

Start targeting the release of the module improvements week by week, this way not only you'll be able to focus on quality but also you'll have more confidence in shipping the full fledged product. 

Don't take everything at once on your head, divide the problem and conquer it. Improve in instalments and be consistent.

**Comment by Harshi7016** [comment]
You are absolutely right, That was the biggest drawback while working on this project.
I should have gone by modules instead of taking on the complete modules.
Rest assured, I have got a good time to focus on this and you have constantly given me great guidance, I will follow your suggestion (y)
